### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by TheaAleksovDev**

### CODE REVIEW

File: src/app/app.component.ts
Comment (valentin-aleksandrov) [inline src/app/app.component.ts:1]: image

File: src/app/components/note/note.component.html
Comment (valentin-aleksandrov) [inline src/app/components/note/note.component.html:1]: image

This button is not a plus on my local.

File: src/app/components/note/note.component.html [hunk 1]
```diff
@@ -4,26 +4,24 @@
...
+        >
+        {{ note.title }}
+      </div>
+      <mat-icon class="delete-icon" (click)="openCloseDeletePopup()"
+        >delete</mat-icon
+      >
+    </h2>
+    <h4 class="note-date" *ngIf="selectedNote === note.id">{{ note.date }}</h4>
+    <p class="note-context" *ngIf="selectedNote === note.id">
+      {{ note.context }}
```
Comment (valentin-aleksandrov) [inline src/app/components/note/note.component.html:24]: Should we also display the **Subject**?

Comment (TheaAleksovDev) [reply src/app/components/note/note.component.html:24]: Why? Isn't it already clear when we look at the bolded subject on the left side?

Comment (TheaAleksovDev) [reply src/app/components/note/note.component.html:24]: Why? Isn't it already clear when we look at the bolded subject on the left side?

Comment (valentin-aleksandrov) [reply src/app/components/note/note.component.html:24]: Apologies, Tea.

File: src/app/components/notes/notes.component.ts [hunk 2]
```diff
@@ -46,12 +46,11 @@ export class NotesComponent implements OnInit {
...
   }
 
-  refreshSubject(subject:string){
-    this.onSelectSubject(subject)
+  refreshSubject(subject: string) {
+    this.onSelectSubject(subject);
   }
-  
+
   anyNotesCheck(): boolean {
```
Comment (valentin-aleksandrov) [inline src/app/components/notes/notes.component.ts]: Is the "anyNotesCheck" method used somewhere?

File: src/app/components/notes/notes.service.ts [hunk 3]
```diff
@@ -7,32 +7,27 @@ export class NotesService {
   [x: string]: any;
```
Comment (valentin-aleksandrov) [inline src/app/components/notes/notes.service.ts]: Why do we need to do this?

**Comment by valentin-aleksandrov** [comment]
image

Do you think it would be useful, if I as an user can delete Subjects?

File: src/app/components/notes/notes.component.ts
Comment (valentin-aleksandrov) [inline src/app/components/notes/notes.component.ts:1]: image

File: src/app/components/notes/notes.component.ts (hunk 2 above)
Comment (TheaAleksovDev) [reply src/app/components/notes/notes.component.ts]: Not really. Removing it.

File: src/app/components/notes/notes.component.ts
Comment (TheaAleksovDev) [reply src/app/components/notes/notes.component.ts:1]: Why are we removing newNoteOpened?

File: src/app/components/notes/notes.service.ts (hunk 3 above)
Comment (TheaAleksovDev) [reply src/app/components/notes/notes.service.ts]: I noticed it while reviewing before pushing and wasn't sure if it was added by TypeScript or if I pasted it while trying to solve a bug. Looks like it doesn't do anything, I'll remove it.

**Comment by valentin-aleksandrov** [comment]
Perhaps you can consider setting up a `min-width` to these items so that you have steady design when the width of the consumer's device is small.
image

File: src/app/components/notes/notes.component.ts
Comment (valentin-aleksandrov) [reply src/app/components/notes/notes.component.ts:1]: Let's just say that I overdid my drawing :D 
The suggestion is only for renaming.

**Review by valentin-aleksandrov — APPROVED** [review]

**Pull request merged**